### PR TITLE
Join Tables PR

### PR DESCRIPTION
**Pull request by fabianofrank**
Join tables

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -57,4 +57,76 @@ FROM animals
...
+FROM visits
+JOIN vets ON vets.id = visits.vets_id
+JOIN animals ON animals.id = visits.animals_id
+ORDER BY visits.date_of_visit asc;
+
+SELECT vets.name, COUNT(visits.date_of_visit)
+FROM visits
+JOIN vets ON vets.id = visits.vets_id
+JOIN animals ON animals.id = visits.animals_id
+WHERE vets.id = (SELECT id FROM vets WHERE name = 'Maisy Smith')
```
Comment (Lordkaito) [inline queries.sql:119]: - [ ] I think you did a good query here but may have misunderstood the question. You are asked to check how many times an animal was seen by a vet that is not specialized in that specie. For example, if the animal is a pokemon, and the vet is specialized in digimon, that counts as one animal with a vet that is not specialized in its specie. So here you are checking for animals that were taken only by Maisy Smith, which has no specialization, but that is not exactly what you were asked. So please work on this issue.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @fabianofrank, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (fabianofrank) [reply queries.sql]: Yes, I understood it wrong, you are correct, thank you for the note!

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @fabianofrank 👋🏻, 

Great job making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

rosycheeks-mochi-peach


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**